src/LinuxTracing/PerfEventReaders.h

```diff
@@ -49,6 +49,18 @@ GenericTracepointPerfEvent ConsumeGenericTracepointPerfEvent(PerfEventRingBuffer
 SchedWakeupPerfEvent ConsumeSchedWakeupPerfEvent(PerfEventRingBuffer* ring_buffer,
                                                  const perf_event_header& header);
 
+SchedWakeupWithCallchainPerfEvent ConsumeSchedWakeupWithCallchainPerfEvent(
```

Comment:
can we take the chance to add `[[nodiscard]]` to all the read methods?

Reply:
+1

Reply:
Done.